README.md

```diff
@@ -9,13 +9,15 @@ You can also find information on how to contribute to this repository, either by
 - [Contributing (general)](#contributing-general)
   - [Quick guide](#quick-guide)
 - [Contributing (songs)](#contributing-songs)
+  - [Format](#format)
+  - [Creating](#creating)
```

Comment:
Should be `modifying` instead of `format` and should be in opposite order

Reply:
Resolved with [f74c3a3](https://github.com/insektionen/songlist/pull/1/commits/f74c3a3ae334e4bc03f38374b34162988bce494f).